radio/src/edgetx.cpp

```diff
@@ -1414,7 +1411,7 @@ void edgeTxInit()
   lcdSetContrast();
```

Comment:
If there is an issue with `BACKLIGHT_ENABLE()` for the pa01, you should be looking at the target specific implementation for it in `radio/src/targets/pa01/board.h` to see what is wrong there, not disabling it for **all** targets.